.github/workflows/scripts/schedule.actions/verifier-installer.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# shellcheck source=/dev/null
+source "./.github/workflows/scripts/e2e-utils.sh"
+
+minimum_version="$MINIMUM_INSTALLER_VERSION"
+list="\"$minimum_version\""
```

Comment:
You use `version_ge` so is it necessary to include `$minimum_version` in the list here? I would think this would duplicate the minimum version.

It also means you can't use a minimum version that doesn't match the tag exactly (e.g. 'v2.5')

Reply:
I have replaced `version_ge` with `version_gt`. Please confirm this is OK.

Reply:
I was thinking the list could just start as empty and you would still pick up all the versions greater than or equal to the minimum version even if the minimum version was something like 'v2.5'.

Reply:
done. thanks